### DISCUSSION TITLE
Safe to go to -dev version and back?

### DISCUSSION THREAD

**Discussion by m0wlheld** [Koenkk/zigbee2mqtt#5808] [General]
For checking, if the -dev version fixes my [recently reported](https://github.com/Koenkk/zigbee2mqtt/discussions/5806) availability issues: Is it safe to try the -dev version of z2m and can I go back to 0.17.0 if things go wrong? Or does -dev version some irreversible updates to database etc., that prevent returning to a previous version? Z2M is running inside a Docker container...

In general: Is -dev more beta or alpha?

**Comment by Koenkk** [comment]
It's safe to switch between 1.17.0 and the current dev version.

-dev is not alpha/beta, it's more a nightly. On every release (1st of the month) a snapshot is taken and this will be the new release.
